charts/queryservice-gateway/values.yaml

```diff
@@ -1,6 +1,7 @@
 replicaCount: 1
 
 platform:
+  responseCacheTTL: 3600
```

Comment:
It would probably make sense to bump the `image.tag` value after adding this; this will help others (or just us when we forget) if we fail to specifically set the image in our helm values and then wonder why we seem to be using an "old" image

Reply:
I didn't even know this has a default value here. Fixed it.